static/src/javascripts/projects/commercial/modules/prebid/prebid.js

```diff
@@ -153,6 +153,24 @@ class PrebidService {
                 () =>
                     new Promise(resolve => {
                         window.pbjs.que.push(() => {
+                            // Capture this specific auction starting
+                            // to set this slot pbaid targeting value
+                            const auctionInitHandler = (data: Object) => {
+                                // Get rid of this handler now.
+                                window.pbjs.offEvent(
+                                    'auctionInit',
+                                    auctionInitHandler
+                                );
+                                advert.slot.setTargeting(
+                                    'pbaid',
```

Comment:
I think there's a general convention that header bidding targeting is called `hb_...`
eg. `hb_bidder` and `hb_ttr`
so perhaps call it `hb_auction`?